src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -112,8 +90,82 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     selectPortfolioCryptoHumanBalanceByFilter(s, feeAssetBalanceFilter),
   )
 
+  const selectedCurrency = useAppSelector(selectSelectedCurrency)
+
+  const getEstimateFeesArgs: () => Promise<EstimateFeesInput> = useCallback(async () => {
+    const amountCryptoBaseUnit = bnOrZero(state?.deposit.cryptoAmount).times(
+      bn(10).pow(asset.precision),
+    )
+    const quote = await getThorchainSaversQuote(asset, amountCryptoBaseUnit)
+
+    return {
+      cryptoAmount: state?.deposit.cryptoAmount ?? '',
+      asset,
+      to: quote.inbound_address,
+      sendMax: false,
+      accountId: accountId ?? '',
+      contractAddress: '',
+    }
+  }, [accountId, asset, state?.deposit.cryptoAmount])
+
+  useEffect(() => {
+    if (!accountId) return
+    ;(async () => {
+      const accountAddress = isUtxoChainId(chainId)
+        ? await getThorchainSaversPosition(accountId, assetId)
+            .then(({ asset_address }) =>
+              chainId === bchChainId ? `bitcoincash:${asset_address}` : asset_address,
+            )
+            .catch(() => '')
+        : ''
+      setMaybeAccountAddress(accountAddress)
+    })()
+  }, [chainId, accountId, assetId])
```

Comment:
If we throw or the current AssetId isn't an UTXO AssetId, this will evaluate to `''`, a falsy value i.e EVM chain adapters will make a no-op of the `from`